frappe/hooks.py

```diff
@@ -261,7 +261,7 @@
 	{
 		'doctype': 'Contact',
 		'match_field': 'email_id',
-		'personal_fields': ['first_name', 'last_name', 'phone', 'mobile_no'],
+		'personal_fields': ['first_name', 'last_name', 'phone'],
```

Comment:
Can we add personal_fields for child table `Contact Phone` and `Contact Address`?

I guess `match_field` might be the problem 🤔

Reply:
added personal_field for contact email, couldnt add anything for contact phone.